test/e2e/storage/volume_provisioning.go

```diff
@@ -513,9 +513,9 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 
 			ginkgo.By(fmt.Sprintf("Checking for residual PersistentVolumes associated with StorageClass %s", class.Name))
 			residualPVs, err = waitForProvisionedVolumesDeleted(c, class.Name)
-			framework.ExpectNoError(err)
 			// Cleanup the test resources before breaking
 			defer deleteProvisionedVolumesAndDisks(c, residualPVs)
+			framework.ExpectNoError(err)
```

Comment:
nit: context

Reply:
I added context. Also, since `ExpectNoError` causes an early return, I removed the unreachable code below.